core/templates/components/state-directives/outcome-editor/outcome-feedback-editor.directive.ts

```diff
@@ -32,11 +33,15 @@ angular.module('oppia').directive('outcomeFeedbackEditor', [
         '/components/state-directives/outcome-editor/' +
         'outcome-feedback-editor.directive.html'),
       controllerAs: '$ctrl',
-      controller: [function() {
+      controller: ['ContextService', function(ContextService) {
         var ctrl = this;
         ctrl.$onInit = function() {
           ctrl.OUTCOME_FEEDBACK_SCHEMA = {
-            type: 'html'
+            type: 'html',
+            ui_config: {
+              hide_complex_extensions: (
+                ContextService.getPageContext() === 'topic_editor')
```

Comment:
Shouldn't this be for both topic and skill editor, since questions can be created in either page.

Reply:
Yep, I realised that later. I'll update the PR. Thanks!

Reply:
btw @aks681, I noticed that in the context service we don't have a "question" context. Do you think we should have one? (I did see one for question_editor but not sure if we use that.)

Reply:
Actually, scratch that, I think ContextService.getEntityType() can be used for this. It has a check for questions.

Reply:
Done. PTAL @aks681!